### PR TITLE
Update server-migration.md

### PR DESCRIPTION
**Pull request by pahuijbr**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/server-migration.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ TIP: you can use the [installer script](server-migration.md#installer script) as
 ## Azure Defender scenarios
 
 ### You're using Azure Defender. The Microsoft Monitoring Agent (MMA) and/or Microsoft Antimalware for Azure (SCEP) are installed and you want to upgrade.
-If you're using Azure Defender, you can leverage the automated upgrade process. See [Protect your endpoints with Security Center's integrated EDR solution: Microsoft Defender for Endpoint](/azure/security-center/security-center-wdatp#enable-the-microsoft-defender-for-endpoint-integration).
+If you're using Azure Defender, you can leverage the automated upgrade process when this enters preview around Q1 of 2022. See [Protect your endpoints with Security Center's integrated EDR solution: Microsoft Defender for Endpoint](/azure/security-center/security-center-wdatp#enable-the-microsoft-defender-for-endpoint-integration) for updates.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/server-migration.md]:
```suggestion
If you're using Azure Defender, you can leverage the automated upgrade process when this enters public preview around the first quarter of 2022. See [Protect your endpoints with Security Center's integrated EDR solution: Microsoft Defender for Endpoint](/azure/security-center/security-center-wdatp#enable-the-microsoft-defender-for-endpoint-integration) for updates. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by pahuijbr** [comment]
LGTM please proceed
________________________________

**Comment by JohanFreelancer9** [comment]
@pahuijbr - could you kindly commit the suggested change? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pahuijbr

**Comment by mjcaparas** [comment]
We will update this page with the availability of that feature once it actually becomes available. Closing this PR for now.

**Pull request closed without merging**